spec/features/users/user_can_delete_their_own_comments_spec.rb

```diff
@@ -18,6 +18,24 @@
     expect(page).to_not have_content("Nice comment")
   end
 
+  it 'they can delete answer comments owned by them' do
```

Comment:
Great testing Dan. Love how using the css selectors is becoming less painful for all of us.